<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitPatchFilesDialog</class>
 <widget class="QDialog" name="GitPatchFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patch Files</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="patchFilesList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="toolTip">
          <string>Press to add patch files to the list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deleteButton">
         <property name="toolTip">
          <string>Press to delete the selected patch files from the list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="upButton">
         <property name="toolTip">
          <string>Press to move the selected file up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="downButton">
         <property name="toolTip">
          <string>Press to move the selected file down</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Strip Count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="stripSpinBox">
       <property name="toolTip">
        <string>Enter number of leading directories to strip off (default 1)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="eofCheckBox">
     <property name="toolTip">
      <string>Select, if the patch has inaccurate end-of-file markers</string>
     </property>
     <property name="text">
      <string>Patch has inaccurate end-of-file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="lineCountsCheckBox">
     <property name="toolTip">
      <string>Select, if the line counts of the patch header may be wrong</string>
     </property>
     <property name="text">
      <string>Don't trust line counts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>patchFilesList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>stripSpinBox</tabstop>
  <tabstop>eofCheckBox</tabstop>
  <tabstop>lineCountsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GitPatchFilesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GitPatchFilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
